manifests/py2wasm/py2wasm@0.1.0.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "py2wasm",
-  "description": "A plugin to convert Python applications to Spin compatible modules",
+  "description": "A plugin to convert Python applications to Spin compatible modules. Note: This plugin has been deprecated in favour of the newer SDK based on componentize-py. To move to the last SDK, uninstall the plugin and visit https://developer.fermyon.com/spin/v2/python-components  ",
```

Comment:
Same typo fixes throughout.  But really, is there much value in retrofitting to old versions?  Is anyone going to install / update to these old versions at this point?  The only scenario I can think of is if someone is on Spin 0.9 or whatever and the more recent version is incompatible, but those users won't be able to run `componentize-py` based components anyway will they?